openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -8,111 +8,17 @@
-        redox_potential: Optional[float] = Field(
-            None, description="Simulation redox potential"
-        )
-
-
-    class BaseForceFieldSettings(SettingsBaseModel, abc.ABC):
-        """Base class for ForceFieldSettings objects"""
-        ...
-
-
-    class OpenMMSystemGeneratorFFSettings(SettingsBaseModel):
-        """Parameters to set up the force field with OpenMM ForceFields
-
-        .. note::
-           Right now we just basically just grab what we need for the
-           :class:`openmmforcefields.system_generators.SystemGenerator`
-           signature. See the `OpenMMForceField SystemGenerator documentation`_
-           for more details.
-
-
-        .. _`OpenMMForceField SystemGenerator documentation`:
-           https://github.com/openmm/openmmforcefields#automating-force-field-management-with-systemgenerator
-        """
-        constraints: Optional[str] = 'hbonds'
-        """Constraints to be applied to system.
-           One of 'hbonds', 'allbonds', 'hangles' or None, default 'hbonds'"""
-
-        rigid_water: bool = True
-        remove_com: bool = False
-        hydrogen_mass: float = 3.0
-        """Mass to be repartitioned to hydrogens from neighbouring
-           heavy atoms (in amu), default 3.0"""
-
-        forcefields: list[str] = [
-            "amber/ff14SB.xml",  # ff14SB protein force field
-            "amber/tip3p_standard.xml",  # TIP3P and recommended monovalent ion parameters
-            "amber/tip3p_HFE_multivalent.xml",  # for divalent ions
-            "amber/phosaa10.xml",  # Handles THE TPO
-        ]
-        """List of force field paths for all components except :class:`SmallMoleculeComponent` """
-
-        small_molecule_forcefield: str = "openff-2.0.0"  # other default ideas 'openff-2.0.0', 'gaff-2.11', 'espaloma-0.2.0'
-        """Name of the force field to be used for :class:`SmallMoleculeComponent` """
-
-        @validator('constraints')
-        def constraint_check(cls, v):
-            allowed = {'hbonds', 'hangles', 'allbonds'}
-
-            if not (v is None or v.lower() in allowed):
-                raise ValueError(f"Bad constraints value, use one of {allowed}")
-
-            return v
-
-
-    class Settings(SettingsBaseModel):
-        """
-        Container for all settings needed by a protocol
-
-        This represents the minimal surface that all settings objects will have.
-
-        Protocols can subclass this to extend this to cater for their additional settings.
-        """
-        forcefield_settings: BaseForceFieldSettings
-        thermo_settings: ThermoSettings
-
-        @classmethod
-        def get_defaults(cls):
-            return cls(
-                forcefield_settings=OpenMMSystemGeneratorFFSettings(),
-                thermo_settings=ThermoSettings(temperature=300 * unit.kelvin),
-            )
-else:
-    from gufe.settings import Settings, SettingsBaseModel  # type: ignore
-
+from gufe.settings import (
```

Comment:
thanks for solving this, this was my nasty lack minute hack to get this working

Reply:
Yeah it was a pain to diagnose but ended up being extremely easy to fix :sweat_smile: